### DISCUSSION TITLE
Chrome flags

### DISCUSSION THREAD

**Discussion by Made0fB1naries** [bromite/bromite#2121] [General]
Idk if everyone of you guys have this problem about chrome flags...
 everytime i go to the flags it just straight up stops bromite or become unresponsive 👀

**Comment by Made0fB1naries** [comment]
https://user-images.githubusercontent.com/86339491/172797850-7145b5b6-12b8-4e56-9bd8-9218616d1842.mp4

It basically just stops

**Reply by uazo** [reply]
try [this](https://github.com/bromite/bromite/issues/1837#issuecomment-1112369868)

**Reply by Made0fB1naries** [reply]
I tried it on google chrome...it actually has the same problem...🤔

**Comment by Made0fB1naries** [comment]
@uazo Because of how unresponsive the Chrome flag is, I'm pretty sure I can't turn off **AImageReader** nor **SurfaceControl**.... While there were some time i got the flag to disable, but relaunching it didn't work.

**Reply by Made0fB1naries** [reply]
Sometimes it says i can relaunch it while sometimes it will just straight up crash

**Comment by Made0fB1naries** [comment]
https://user-images.githubusercontent.com/86339491/176389936-8fe3f438-cc93-4027-8847-efaeb8625dd9.mp4

**Reply by Made0fB1naries** [reply]
This is v102.0.5005.96
